### DISCUSSION TITLE
Uncaught Error: Module parse failed: Unexpected token (2013:23)

### DISCUSSION THREAD

**Discussion by Ademir-Oliveira-Dev** [twilio-professional-services/flex-project-template#597] [General]
Someone any time see this error when try to run the template locally? I follow the steps in the docs and I try to run "npm start" and I see this error:
Uncaught Error: Module parse failed: Unexpected token (2013:23)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     unregister() {
|         document.removeEventListener("click", this.handleClickOutside);
>         this.darkMedia?.removeEventListener("change", this.darkMediaCallback);
|         this.unobserve();
|     }

**Comment by dremin** [comment]
Which version of the Flex plugins CLI do you have installed?
